pkg/plugins/manager/manifest.go

```diff
@@ -205,7 +206,37 @@ func getPluginSignatureState(log log.Logger, plugin *plugins.PluginBase) (plugin
 		var unsignedFiles []string
 		for _, f := range plugin.Files {
 			if _, exists := manifestFiles[f]; !exists {
-				unsignedFiles = append(unsignedFiles, f)
+				fullPath, err := filepath.Abs(filepath.Join(plugin.PluginDir, f))
```

Comment:
It does look a bit weird that `manifestFiles` haven't handled symlinked files earlier when scanning plugin files, but I probably overlook some difficulties with doing that?

Reply:
`manifestFiles` wouldn't have any simlink entries as the signing process itself will ignore them - but `plugin.Files` potentially will. You just gave me an idea for making this a lot simpler though! ❤️